### DISCUSSION TITLE
What is the point of fallback resolvers?

### DISCUSSION THREAD

**Discussion by datainvestor** [mirumee/ariadne#639] [Q&A]
I have followed Flask+GraphQL tutorial accesible here:
https://www.apollographql.com/blog/graphql/python/complete-api-guide/

And in one case they imported snake_case_fallback_resolvers and set it in the make_executable_schema function. Now in Ariadne docs its described as something that makes default resolver to a schema without anything? How does exactly this work?
So if I create schema in schema.graphql file, then it will automatically create a resolver for it so that I can access it in graphql playground? Im not sure if I am getting it right. I couldnt make a query to work without any resolver so maybe Im missing something.

**Comment by nilansaha** [comment]
I believe the idea in this specific case is to bridge the gap between language naming conventions. For example languages like java or js like to use camel case like `postQuery` while languages like Python like to use snake case like `post_query`. Usually graphql schemas are written in camel case so to make that more pythonic we have a snake_case_fallback_resolvers. What it does is the app will first check if there is a resolver set for `postQuery` and if there is not it will try to fall back to `post_query`. Hope that makes sense
